### PR TITLE
Update install and running to encourage people to use binaries

### PR DESCRIPTION
**Pull request by fsidhoum**

### CODE REVIEW

File: testnet/install.rst [hunk 1]
```diff
@@ -8,8 +8,8 @@ Installing a node
 
     Right now 4 cores and 8 GB of RAM should be enough to run a node, but it might increase in the future. More info in the :ref:`FAQ <testnet-faq>`.
 
-From binaries
-=============
+From binaries (recommanded way)
```
Comment (AurelienFT) [inline testnet/install.rst:11]:
```suggestion
From binaries (simple installation)
```

File: testnet/install.rst [hunk 2]
```diff
@@ -18,8 +18,8 @@ can simply download the latest binary below and go the the next step: :ref:`Runn
 - `Linux binary <https://github.com/massalabs/massa/releases/download/TEST.19.1/massa_TEST.19.1_release_linux.tar.gz>`_ - only works with libc2.28 at least (for example Ubuntu 20.04 and higher)
 - `MacOS binary <https://github.com/massalabs/massa/releases/download/TEST.19.1/massa_TEST.19.1_release_macos.tar.gz>`_
 
-From source code
-================
+From source code (for advanced users)
```
Comment (AurelienFT) [inline testnet/install.rst:21]:
```suggestion
From source code (advanced installation)
```

File: testnet/running.rst [hunk 3]
```diff
@@ -4,8 +4,8 @@
 Running a node
 ==============
 
-From binaries
-=============
+From binaries (recommanded way)
```
Comment (AurelienFT) [inline testnet/running.rst:7]:
```suggestion
From binaries (simple installation)
```

File: testnet/running.rst [hunk 4]
```diff
@@ -53,8 +53,8 @@ Then:
 
 Replace <PASSWORD> with a password that you will need to keep to restart your client
 
-From source code
-================
+From source code (for advanced users)
```
Comment (AurelienFT) [inline testnet/running.rst:56]:
```suggestion
From source code (advanced installation)
```

**Comment by AurelienFT** [comment]
@aoudiamoncef what do you think of this ?

**Review by aoudiamoncef — COMMENTED** [review]
IMHO, it's a coherent approche, with this advices we got the good from normal and advanced users. it's a yes

**Review by AurelienFT — APPROVED** [review]

**Pull request merged**